docs/fuzzy.rst

```diff
@@ -5,7 +5,7 @@ Fuzzy attributes
 
 .. note:: Now that FactoryBoy includes the :class:`factory.Faker` class, most of
           these built-in fuzzers are deprecated in favor of their
-          `Faker <http://www.joke2k.net/faker/>`_ equivalents. Further
+          `Faker <http://faker.rtfd.org>`_ equivalents. Further
```

Comment:
`https://faker.readthedocs.io/` might be the better place to link to, since that's where rtfd.org redirect to (and TLS is supported).